src/main/resources/edu/cornell/kfs/coa/cu-spring-coa.xml

```diff
@@ -192,4 +190,12 @@
 		class="edu.cornell.kfs.coa.service.impl.GlobalObjectWithIndirectCostRecoveryAccountsServiceImpl">	
     </bean>
 
+    <bean id="organizationReversionFiscalYearMaker" parent="FiscalYearMaker"
+          p:fiscalYearOneBehind="false">
+    </bean>
+
+    <bean id="organizationReversionDetailFiscalYearMaker" parent="FiscalYearMaker"
```

Comment:
It looks like neither the org-reversion FY maker nor the org-reversion-detail FY maker are using the "parentBean" convention in base code. Thus, you will need to add the full contents of both beans to this file, and then modify their properties accordingly.